sql/changes/sc.04.08.2000.sql

```diff
@@ -0,0 +1,13 @@
+-- django_hstore's implementation of hstore always sets the field to something,
+-- so that whether the field is null or not is irrelevant. This is bad django,
+-- bad sql, and bad python.
+--
+-- django.contrib.postgres's implementation does not do this, but we send in
+-- nulls all the time, so alter tables accordingly.
+
+ALTER TABLE netbox
+    ALTER COLUMN data DROP NOT NULL;
```

Comment:
If you do this, you first need to make sure that _existing_ NULL values are changed _first_, or someone will have a bad day when upgrading their NAV.

Reply:
There cannot be any existing NULLs since the columns are NOT NULL.

On the other hand, if we always set `default={}`, NULL versus NOT NULL is irrelevant. They could all be NULL, or all be NOT NULL, it would only make a difference for non-django 
access/connection methods.

Reply:
I also just realized, you need to name this script in the `sc.04.09.xxxx` range, since this will not be merged to the 4.8.x branch, but will be part of the 4.9 release.

Reply:
`sc.04.09.0001.sql` ? That is, first migration of that version?

Reply:
Yes, but there are pull requests in-flight that also add to the `sc.04.09.xxxx` series, so beware of potential overlaps (first come, first served :D)

Reply:
Instead of setting not null, I'm setting default empty instead. This way, all usage of the database will behave the same way. A variant of the paranoia option. https://github.com/UNINETT/nav/pull/1684/commits/e0a749766755b12dd0a6d8d29014730a5edc36c9